File: CodeQuestAcademy/src/NotSoSelfDriving.java
import java.util.Scanner;
// Problem B: Not So Self-Driving
// Author: Chris Liu, California High School
// Difficulty: Practice
// Code Written By: Sam L.
public class NotSoSelfDriving {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T = sc.nextInt();
        for (int i = 0; i < T; i++)
        {
            String s = sc.next();
            String[] arr = s.split(":");
            double a = Double.parseDouble(arr[0]);
            double b = Double.parseDouble(arr[1]);
            if (b / a <= 1)
                System.out.println("SWERVE");
            else if (b / a <= 5)
                System.out.println("BRAKE");
            else
                System.out.println("SAFE");
        }
    }
}

File: CodeQuestAcademyC++/Practice/NotSoSelf-Driving.cpp
/*
Problem B: Not So Self-Driving
Practice Problem
Author: Chris Liu, California High School
*/

#include <bits/stdc++.h>
#define lli long long int
using namespace std; 

lli T;
double V, X;
string S;

int main()
{
	cin >> T;
	for (lli t = 0; t < T; t++) {
		cin >> S;
		V = stod(S.substr(0, S.find(":")));
		X = stod(S.substr(S.find(":") + 1));
		if (X / V <= 1)
			cout << "SWERVE" << endl;
		else if (X / V <= 5)
			cout << "BRAKE" << endl;
		else
			cout << "SAFE" << endl;
	}
}

/*
Sample Input:
5
23.15:98.34
2.40:17.33
6.79:5.01
0.00:1.53
113.56:113.56
*/

/*
Sample Output:
BRAKE
SAFE
SWERVE
SAFE
SWERVE
*/
